jobs/JREGIONAL_RUN_PYTHON_GRAPHICS

```diff
@@ -149,6 +153,38 @@ case ${GRAPHICS_TYPE} in
       --max_plev 100"
     ;;
 
+  "enspanel")
+
+    file_tmpl="${NET}.t${CDATE:8:2}z.bgdawpf{FCST_TIME:03d}.tm00.grib2"
+    file_type=prs
+    extra_args="\
+      --tiles $tiles \
+      --images ${PYTHON_GRAPHICS_DIR}/image_lists/rrfs_ens.yml hourly"
+
+    # Link in control member to the location it should be
```

Comment:
The control member is linked into a similar location as all the other members as member 0 such that it matches the appropriate template passed to pygraf.